readme.md

```diff
@@ -292,6 +292,13 @@ Default: `true`
 
 Whether to allow unknown flags or not.
 
+##### preferNestedFlags
+
+Type `boolean`\
+Default: `false`
+
+This option makes sense if your CLI may be invoked inside another one. For example, `npx -p foo -p bar bar-cli -- --opts-for='bar-cli'`.
```

Comment:
**Any** CLI may be invoked inside another one. I really don't think this is the best way to resolve your problem.

Reply:
My issue may be related to npx: seems like it loads my CLI in its own process. I'll check the `process.argv[1]` value.